lib/entities/request/ClassicRequest.ts

```diff
@@ -23,6 +23,7 @@ export interface ClassicConfig {
   forceMixedRoutes?: boolean;
   debugRoutingConfig?: string;
   unicornSecret?: string;
+  quoteSpeed?: string;
```

Comment:
can you also add to joi? https://github.com/Uniswap/unified-routing-api/blob/f24020dcadf956016e959ef6a325ac2511200f77/lib/util/validator.ts#L92

Reply:
Added, thanks for the quick review!